frr-metrics/exporter.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	stdlog "log"
+	"net/http"
+	"os"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/prometheus/exporter-toolkit/web"
+
+	"go.universe.tf/metallb/frr-metrics/collector"
+	"go.universe.tf/metallb/internal/logging"
+)
+
+var (
+	metricsPort   = flag.String("metrics-port", ":7473", "Address to listen on for web interface.")
```

Comment:
Does this mean that `metricsPort` needs to be specified as `:<port_number`? Maybe change this to `flag.Uint`?

Reply:
done